packages/modelserver-node/src/routes/index.ts

```diff
@@ -0,0 +1,11 @@
+/********************************************************************************
+ * Copyright (c) 2022 STMicroelectronics.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0, or the MIT License which is
+ * available at https://opensource.org/licenses/MIT.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR MIT
+ *******************************************************************************/
+export * from './route-utils';
```

Comment:
is anybody using this?

Reply:
Not yet, but if an adopter wants to re-use the validation wrapper hook, the import path is cleaner that way